tests/forms_tests/contrib/storages.py

```diff
@@ -0,0 +1,7 @@
+from django.contrib.staticfiles import storage
```

Comment:
I think I would be more inclined to put these tests in `staticfiles_tests/test_forms.py` (new file).